docs/configuration.rst

```diff
@@ -76,6 +76,9 @@ ACCOUNT_SIGNUP_FORM_CLASS (=None)
 ACCOUNT_SIGNUP_PASSWORD_VERIFICATION (=True)
   When signing up, let the user type in their password twice to avoid typo's.
 
+ACCOUNT_SIGNUP_EMAIL_VERIFICATION (=False)
+  Same as above, but for an email.
```

Comment:
I need to sort these settings alphabetically some time, to avoid issues: "When signing up, let the user type in their email twice to avoid typo's."